tests/prague/eip7692_eof_v1/eip7620_eof_create/test_eofcreate_failures.py

```diff
@@ -168,7 +168,11 @@ def test_initcode_aborts(
         pytest.param(MAX_BYTECODE_SIZE + 1, id="overmax"),
         pytest.param(MAX_INITCODE_SIZE - factory_size, id="initcodemax"),
         pytest.param(MAX_INITCODE_SIZE - factory_size + 1, id="initcodeovermax"),
-        pytest.param(0xFFFF - factory_size, id="64k-1"),
+        pytest.param(
+            0xFFFF - factory_size,
+            id="64k-1",
+            marks=pytest.mark.skip("Oversized container in pre-alloc"),
```

Comment:
Should the prior line get this skip mark as well?  There is a pending spec change to set max container sizes and initcodemax is a front runner.

Reply:
Yes, and actually, shouldn't be using a container creating tx a better place to put these larger container tests? Maybe have both cases ideally.

Reply:
Added in the last commit, also added the function signature for the same test but using a create tx.